noetic-foxy/init-env.sh

```diff
@@ -40,16 +40,12 @@ export PATH=$PATH:/usr/local/cuda/bin
 export GENICAM_GENTL32_PATH=$GENICAM_GENTL32_PATH:"/opt/Vimba_5_0/VimbaGigETL/CTI/x86_32bit/"
 export GENICAM_GENTL64_PATH=$GENICAM_GENTL64_PATH:"/opt/Vimba_5_0/VimbaGigETL/CTI/x86_64bit/"
 
-if [ -f "/opt/ros/noetic/setup.bash" ]; then
-    source /opt/ros/noetic/setup.bash
-fi
-
-if [ -f "/opt/autoware.ai/ros/install/setup.bash" ]; then
-    source /opt/autoware.ai/ros/install/setup.bash
-fi
-
 if [ -f "/opt/carma/install/setup.bash" ]; then
```

Comment:
I think this will still end up sourcing ROS1. 
ROS2 is sourced by `source /opt/carma/install_ros2/setup.bash`
This image has both ROS1 and ROS2, and it is okay to default to ROS1. 

It is up to the user of this image to correctly configure to use ROS1 or ROS2.

Reply:
About this some of the newer repos don't have any ros1 components so they install ros2 only code under /opt/carma/install, which can cause the paths to mix up.

Reply:
Ah right. Okay yeah I think I am okay with this because opt/carma/install should have autoware.ai as well as noetic (or even foxy) sourced